packages/lib-classifier/src/components/Classifier/ClassifierContainer.js

```diff
@@ -96,6 +111,10 @@ export default function ClassifierContainer({
     i18n.changeLanguage(locale)
   }
 
+  /*
+    Make sure the classifierStore's active project is in sync with parent app.
+    Useful for standalone classifier such as the dev classifier.
+  */
```

Comment:
Sorry, I still don't understand this comment. Lines 118–127 store the project snapshot (from the classifier props) in the classifier store and make it active. Otherwise, I think the classifier won't run at all if there isn't an active project. 

TBH I think store hydration might be easier to understand if we used `getSnapshot` and `applySnapshot` to set up the initial classifier state.

Reply:
Gotcha. I was a little confused too, but now that I understand how workflow storage is happening in the lines below this, I'll update this block's comment similarly.